Rematch/Addon.lua

```diff
@@ -367,12 +396,122 @@ function RematchPlugin:UpdateDBRematch4To5(convertedTeams)
     for teamID, script in self:IterateScripts() do
         if not self.savedRematchTeams[teamID] then
             C_Timer.After(0.9, function()
-                GUI:Notify{
+                GUI:Notify({
                     text = format('%s\n|cff00ffff%s|r', ns.L.ADDON_NAME, format(ns.L.SELECTOR_REMATCH_4_TO_5_UPDATE_ORPHAN, script:GetName(), teamID)),
                     icon = ns.ICON,
                     duration = -1,
-                }
+                })
+            end)
+        end
+    end
+end
+
+local section           = 'PET BATTLE SCRIPT'
+local patternSeps       = '%-%-%-%-%-'
+local outputSeps        = '-----'
+local patternBegin      = patternSeps .. 'BEGIN ' .. section .. patternSeps
+local outputBegin       = outputSeps  .. 'BEGIN ' .. section .. outputSeps
+local patternEnd        = patternSeps .. 'END '   .. section .. patternSeps
+local outputEnd         = outputSeps  .. 'END '   .. section .. outputSeps
+local patternScriptNote = '\n*' .. patternBegin .. '\n(.*)' .. patternEnd .. '\n*'
+local outputScriptNoteB = '\n'  .. outputBegin .. '\n'
+local outputScriptNoteA =                             '\n'  .. outputEnd .. '\n'
```

Comment:
Are these strings worth keeping in a separate file (or at the top of this one) just for easy access in case we need to change them?

Reply:
Don't think so, not like we would need them anywhere else. Even assembling them like this already is overkill.